src/components/RP_Youtube.component

```diff
@@ -0,0 +1,94 @@
+<apex:component controller="RP_YoutubeController">
+
+    <h2 class="slds-text-heading_medium"> {!$Label.RP_ProductNameLabel} {!$Label.RP_Videos}</h2>
+
+    <div class="slds-box slds-m-top_medium slds-p-top_none youtube-section"  >
+        <div class="slds-grid slds-wrap slds-grid_pull-padded" >
+
+            <div id="selectedVideo" class="slds-p-horizontal_small slds-size_9-of-12 video-wrapper">
+
+                <div class="video-list" data-tooltip="{!youtubeMainVideo.videoTitle}" />
+
+            </div>
+
+             <div id="videos" class="slds-p-horizontal_small slds-size_3-of-12" >
+
+                <apex:repeat value="{!youtubeVideosData}" var="video" id="theVideos">
+
+                    <div class="video-list" data-tooltip="{!video.videoTitle}">
+
+                        <img id="{!video.videoId}" src="{!video.videoThumbnail}" class='video-thumbnail' alt="{!video.videoTitle}"/>
+
+                    </div>
+
+                    <hr class='videos-hr' />
+
+                </apex:repeat>
+                
+            </div>
+
+            <div class="slds-p-horizontal_small slds-size_1-of-1 slds-large-size_3-of-3 large-column" >
+
+                <div class="slds-clearfix">
+
+                    <div class="slds-float_right">
+
+                        <apex:outputLink target="_blank" value="https://www.youtube.com/{!JSENCODE(youtubeChannelName)}" styleClass="links">
+                            {!$Label.RP_YoutubeChannel}
+                        </apex:outputLink>
+
+                    </div>
+
+                </div>
+
+            </div>
+
+        </div>
+    </div>
+
+    <apex:includeScript value="{!URLFOR($Resource.RP_Resources, 'Resources/js/loadMainVideo.js')}"/>
```

Comment:
as mentioned earlier, we want the sources for the static resources added to /StaticResourceSources in the project.